eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -1311,6 +1322,9 @@ registerPool = do
 withNoFixup :: ImpTestM era a -> ImpTestM era a
 withNoFixup = local $ iteDoTxFixupL .~ False
 
+withPostFixup :: (Tx era -> ImpTestM era (Tx era)) -> ImpTestM era a -> ImpTestM era a
```

Comment:
Not too happy with the name, because it's in essence also fixup, but it's "manual" rather than "automatic" as opposed to `fixupTx`

Reply:
Would `withPostFixupHook` be better?

Reply:
Not sure if "hook" makes it any better.. I think between "withPostFixup" and "withPostFixupHook" I prefer the former, because we haven't used the "hook" terminology so far, even though I guess "withNoFixup" is also a hook?  So it feels somehow inconsistent.